### PR TITLE
new: board: nRF52 mini development

### PR DESCRIPTION
**Pull request by d00616**
This change is for a nameless development board available via aliexpress. https://github.com/RIOT-OS/RIOT/wiki/Board:-nRF52-minidev

### CODE REVIEW

File: boards/nrf52-minidev/Makefile.include [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Define the flash-tool and default port:
+export PROGRAMMER ?= openocd
+
+ifeq ($(PROGRAMMER),openocd)
+else ifeq ($(PROGRAMMER),jlink)
+	# setup JLink for flashing
+	export JLINK_DEVICE := nrf52
+	# special options when using SoftDevice
+	ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
+		export JLINK_PRE_FLASH := loadfile $(BINDIR)/softdevice.hex
```
Comment (kaspar030) [inline boards/nrf52-minidev/Makefile.include]: you might want to prepend "erase\n" here as needed for nrf52dk.

File: boards/nrf52-minidev/include/arduino_board.h [hunk 2]
```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_
```
Comment (kaspar030) [inline boards/nrf52-minidev/include/arduino_board.h]: missing sth?

File: boards/nrf52-minidev/include/arduino_board.h [hunk 3]
```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2015 Freie Universität Berlin
```
Comment (kaspar030) [inline boards/nrf52-minidev/include/arduino_board.h]: 2015?

File: boards/nrf52-minidev/include/board.h [hunk 4]
```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2016 Feie Universität Berlin
```
Comment (kaspar030) [inline boards/nrf52-minidev/include/board.h]: Feie -> Freie

File: boards/nrf52-minidev/board.c [hunk 5]
```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ * Copyright (C) 2016 Frank Holtz
```
Comment (kaspar030) [inline boards/nrf52-minidev/board.c]: here and below: we usually add an email address.

File: boards/nrf52-minidev/Makefile.include (hunk 1 above)
Comment (d00616) [inline boards/nrf52-minidev/Makefile.include]: The jlink stuff is taken from nrf52dk. I cannot check the functionality with a jlink adapter.

Comment (kaspar030) [inline boards/nrf52-minidev/Makefile.include]: Pls see #5893.

**Comment by jfbortolotti** [comment]
@d00616 can you provide more info or a link on this nRF52 minidev board ? I have not been able to find it on Aliexpress :(. Thanks.

**Comment by d00616** [comment]
This Board is listed when you search for "nrf52832".


Am 03.10.2016 um 23:40 schrieb jfbortolotti:

**Comment by jfbortolotti** [comment]
Got it ! Thanks d00616.

File: boards/nrf52-minidev/include/arduino_board.h (hunk 3 above)
Comment (d00616) [inline boards/nrf52-minidev/include/arduino_board.h]: This file and other files are taken from nrf52-dk board with minimal changes.

File: boards/nrf52-minidev/Makefile.include (hunk 1 above)
Comment (d00616) [inline boards/nrf52-minidev/Makefile.include]: Fixed.

**Comment by OlegHahm** [comment]
Anyone still working on support for this board?

**Comment by d00616** [comment]
Sorry, I have no plans to integrate this into RIOT in the future. This board comes from Shenzhen Taida Century Technology, so a name can given.

**Comment by tcschmidt** [comment]
Closed because author discontinued work on this,

**Pull request closed without merging**